metrics/handler.go

```diff
@@ -55,10 +54,10 @@ func (m *Metrics) NewLatencyMetricsHandler(h http.Handler) http.Handler {
 		latency := time.Since(start).Seconds()
 
 		header := lrw.Header()
-		log.Debugf("Headers after processing: %v", header)
+		// log.Debugf("Headers after processing: %v", header)
```

Comment:
We should re-enable, remove or put a comment explaining why they're commented out

Reply:
removed